app/core/Encryptor/pbkdf2.test.ts

```diff
@@ -0,0 +1,75 @@
+import { stringToBytes } from '@metamask/utils';
+import { pbkdf2 } from './pbkdf2';
+import { NativeModules } from 'react-native';
+
+const mockPassword = 'mockPassword';
+const mockSalt = '00112233445566778899001122334455';
+
+describe('pbkdf2', () => {
+  afterEach(() => {
+    jest.restoreAllMocks();
+  });
+  it('should use the native implementation of pbkdf2 with main aes', async () => {
```

Comment:
Cosmetic: Just to separate "logical blocks"

```suggestion
  
  it('should use the native implementation of pbkdf2 with main aes', async () => {
```